webapp/channels/src/sass/components/_videos.scss

```diff
@@ -2,72 +2,132 @@
 @use "utils/variables";
 
 .post {
+    // Container for both video and thumbnail
     .video-div {
         position: relative;
-        overflow: hidden;
+        overflow: hidden; // Ensure hover effects stay contained
+        width: 100%;
         max-width: 480px;
-        height: 360px;
```

Comment:
I don't think it's safe to remove the fixed height from here since it'll likely lead to scroll pop/content shift because the browser won't know the height of the video when the app first loads. Ideally, I think we'd set a fixed height for videos in MM (instead of fixing an aspect ratio) and then we'd scale the width based on that since that should work outside of maybe mobile view, although we could probably get away with setting a fixed height for the thumbnail and then rescaling it when the video itself opens

Reply:
added max-height